dataplane/worker_gc.cpp

```
@@ -940,6 +949,17 @@ inline bool worker_gc_t::is_timeout(const uint16_t timestamp,
 	return ((uint16_t)(current_time - timestamp) > timeout);
 }
 
+inline uint16_t worker_gc_t::get_acl_tcp_timeout(const uint8_t flags, const uint32_t ack_timeout, 
+																								 const uint32_t syn_timeout, const uint32_t fin_timeout) {
+	if (flags & (RTE_TCP_ACK_FLAG | RTE_TCP_SYN_FLAG)) {
+		return ack_timeout;
+	}
+	if (flags & RTE_TCP_SYN_FLAG) {
+		return syn_timeout;
+	}
+	return fin_timeout;
```

Seems strange to see FIN timeout as a "default" value. Maybe it's reasonable to create а real default timeout, while processing FIN timeout explicitly using mask?